### PR TITLE
Redesign about page

### PR DESCRIPTION
**Pull request by revengemiroz**
1. Changed the card in contributors page
2. Change the CSS styles
3. Change the layout height to 100vh
3. Change the font to use Poppins font
4. open the link of contributors in a new tab

### CODE REVIEW

File: apps/web/src/components/Member/styles.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { styled } from 'stitches/stitches.config';
```
Comment (ivanms1) [inline apps/web/src/components/Member/styles.js]: make this a .ts file

File: apps/web/src/pages/About/About.tsx [hunk 2]
```diff
@@ -6,8 +6,8 @@ import { StyledAbout } from './styles';
 
 const About = () => (
   <StyledAbout>
-    <Members />
     <NextSeo title='About Project Shelf' />
+    <Members />
```
Comment (ivanms1) [inline apps/web/src/pages/About/About.tsx]: move back above the seo tag

File: apps/web/src/pages/About/styles.ts [hunk 3]
```diff
@@ -1,10 +1,7 @@
 import { styled } from '@/stitches/stitches.config';
 
 export const StyledAbout = styled('div', {
-  maxWidth: '100%',
-  margin: 'auto',
-  textAlign: 'center',
-  paddingLeft: '2em',
-  paddingRight: '2em',
-  backgroundColor: 'whitesmoke',
+  backgroundColor: '#f2f3fc',
```
Comment (ivanms1) [inline apps/web/src/pages/About/styles.ts]: add color to variables please

File: apps/web/tsconfig.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": "tsconfig/nextjs.json",
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "**/*.js"],
```
Comment (ivanms1) [inline apps/web/tsconfig.json]: remove this, no js files

**Review by ivanms1 — CHANGES_REQUESTED** [review]

File: apps/web/src/components/Member/styles.js (hunk 1 above)
Comment (revengemiroz) [reply apps/web/src/components/Member/styles.js]: done

File: apps/web/src/pages/About/styles.ts (hunk 3 above)
Comment (revengemiroz) [reply apps/web/src/pages/About/styles.ts]: done

File: packages/ui/Modal/styles.ts [hunk 5]
```diff
@@ -27,6 +27,7 @@ export const overlay = css({
   '& :global(.ReactModal__Overlay--before-close)': {
     opacity: 0,
   },
+  length: 0,
```
Comment (ivanms1) [inline packages/ui/Modal/styles.ts]: please remove this

File: apps/web/pages/styles.css [hunk 6]
```diff
@@ -1,11 +1,12 @@
+@import url("https://use.typekit.net/uhm1qvw.css");
```
Comment (ivanms1) [inline apps/web/pages/styles.css]: what is this?
please import the font from google fonts
@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');

File: packages/ui/Modal/styles.ts [hunk 7]
```diff
@@ -46,4 +47,5 @@ export const modal = css({
   '&:focus': {
     outline: 0,
   },
+  length: 0,
```
Comment (ivanms1) [inline packages/ui/Modal/styles.ts]: same

File: packages/ui/stitches.config.ts [hunk 8]
```diff
@@ -1,4 +1,4 @@
-import { createStitches } from '@stitches/react';
```
Comment (ivanms1) [inline packages/ui/stitches.config.ts]: remove prettier changes

**Review by ivanms1 — COMMENTED** [review]
last few changes :)

File: apps/web/pages/styles.css (hunk 6 above)
Comment (revengemiroz) [reply apps/web/pages/styles.css]: done

**Review by ivanms1 — APPROVED** [review]
LGTM

**Pull request merged**